tests/Console/Report/FixReport/CheckstyleReporterTest.php

```diff
@@ -38,11 +38,20 @@ final class CheckstyleReporterTest extends AbstractReporterTestCase
 
     public static function setUpBeforeClass(): void
     {
-        self::$xsd = file_get_contents(__DIR__.'/../../../../doc/schemas/fix/checkstyle.xsd');
+        parent::setUpBeforeClass();
+
+        $content = file_get_contents(__DIR__.'/../../../../doc/schemas/fix/checkstyle.xsd');
+        if (false === $content) {
+            throw new \RuntimeException('Cannot read file.');
+        }
+
+        self::$xsd = $content;
     }
 
     public static function tearDownAfterClass(): void
     {
+        parent::tearDownAfterClass();
```

Comment:
Parent method is empty, this is superfluous. Same for `setUpBeforeClass`.

Reply:
I will have no clue to call the parent method in child class if i only modify parent method. or if parent method is modified of PHPUnit level.

with that, whenever we override PHPUnit's method, i want to build on top, not to replace PHPUnit method